src/main/java/seedu/commands/listcommands/InvalidListCommand.java

```diff
@@ -0,0 +1,20 @@
+package seedu.commands.listcommands;
+
+import seedu.classes.Ui;
+import seedu.commands.Command;
+import seedu.type.IncomeList;
+import seedu.type.SpendingList;
+
+public class InvalidListCommand extends Command {
+
+    private final Exception exception;
+
+    public InvalidListCommand(Exception e) {
```

Comment:
It may be more specific to use `WiagiInvalidInputException` as the parameter, instead of `Exception`.